src/main/java/com/programmers/ticketparis/member/service/CustomerService.java

```diff
@@ -34,6 +35,13 @@ public CustomerIdResponse createCustomer(CustomerCreateRequest customerCreateReq
         return CustomerIdResponse.from(customerId);
     }
 
+    public CustomerResponse login(CustomerLoginForm customerLoginForm) {
+        return customerRepository.findByUsername(customerLoginForm.getUsername())
+            .filter(m -> m.checkPassword(customerLoginForm.getPassword()))
```

Comment:
`p2`

- `m`으로 해두면 딱 보았을 때 명시적으로 뭔지 알아보기 어려울 것 같다는 생각이 드네요! 아래와 같이 명시적으로 customer 이런식으로 적는건 어떻게 생각하세요?

```suggestion
            .filter(customer -> customer.checkPassword(customerLoginForm.getPassword()))
```

Reply:
넵 변수명 명시하는 편이 훨씬 가독성이 살아날 것 같네요!! 반영했습니다~!
7ef6f38a2bba72baf8018c54eb489fdde0a95346